### PR TITLE
Style and Temp file closing warning fix

### PR DESCRIPTION
**Pull request by marcharper**
Re #681

### CODE REVIEW

**Comment by drvinceknight** [comment]
Have opened https://github.com/marcharper/Axelrod/pull/4 with a fix for the sphinx build.

File: axelrod/tournament.py [hunk 1]
```diff
@@ -111,6 +111,8 @@ def play(self, build_results=True, filename=None,
         if build_results:
             return self._build_result_set(progress_bar=progress_bar,
                                           keep_interactions=keep_interactions)
+        else:
```
Comment (drvinceknight) [inline axelrod/tournament.py]: Hallelujah!

**Comment by drvinceknight** [comment]
This looks great. Out of curiosity, did you use some automatic thing to do this or did you work through all the files yourself (Herculean effort it that's how you did it!)? 👍

**Comment by marcharper** [comment]
Just the old fashioned way!

**Comment by drvinceknight** [comment]
> Just the old fashioned way!

👍 💪 

---

This PR looks good to me. Nice clean up of things!

**Comment by marcharper** [comment]
@drvinceknight Two more commits, mostly cleaning up unused parameters for the tournament classes.

**Comment by drvinceknight** [comment]
:+1:

**Comment by drvinceknight** [comment]
(Assuming Travis is happy with the sphinx build.)

**Pull request merged**